Launcher/Classes/ViewControllers/InspectorViewController.swift

```diff
@@ -408,9 +409,23 @@ extension InspectorViewController: NSOutlineViewDelegate {
         let selectedIndex = outlineView.selectedRow
         
         guard let feedItem = outlineView.item(atRow: selectedIndex) as? String else { return }
-        let filePath4 = "/tmp/localized.txt"
         localizedTextField.stringValue = ""
-        try? fileManager.removeItem(atPath: filePath4)
         elementTextField.stringValue = feedItem
+        
+        CommandExecutor(launchPath: Constants.FilePaths.Bash.checkDuplicates ?? "", arguments: [feedItem]).execute()
+        waitingForFile(withName: self.cloneInfoPath, numberOfRetries: 30, enableSpinner: false) {
+            guard let streamReader = StreamReader(path: self.cloneInfoPath) else { return }
+            defer {
+                streamReader.close()
+                try? self.fileManager.removeItem(atPath: self.cloneInfoPath)
+            }
+            guard let urlLine = streamReader.nextLine(), urlLine == "false" else { return }
+            SharedElement.shared.stringValue = feedItem
+            DispatchQueue.main.async {
+                self.cloneButton.isHidden = false
+                self.cloneLabel.stringValue = "The Element is not unique"
```

Comment:
Can you make sure this string is localized (`.localized`)? That case, if we localize this at some point, we won't miss this string.